docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-yp-issues.md

```diff
@@ -64,3 +64,23 @@ do
   ssh $IP firewall-cmd --zone=public --add-port=6379/tcp;
 done
 ```
+
+## Configure load balancer for Helm charts
+
+If you experience issues with configuring a load balancer when installing and configuring YugabyteDB Anywhere on Kubernetes, you should define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved.
+
+For Amazon Web Services (AWS) load balancers, the default service configuration is adequate; however, if customization is required, the backend protocol should be configured to HTTP by setting the following AWS load balancer controller annotations:
```

Comment:
This could be a separate section or bullet point different from the above perhaps?

Reply:
It should be fine as is.

Reply:
Formatted as a list. Please take a look.